### PR TITLE
Move the AI to Mods.Common

### PR DESCRIPTION
**Pull request by penev92**
Currently based on #7435

### CODE REVIEW

File: OpenRA.Mods.RA/OpenRA.Mods.RA.csproj [hunk 1]
```diff
@@ -213,9 +187,6 @@
   -->
   <PropertyGroup>
     <PostBuildEvent>mkdir "$(SolutionDir)mods/ra/"
-copy "$(TargetPath)" "$(SolutionDir)mods/ra/"
-cd "$(SolutionDir)thirdparty/"
-copy "FuzzyLogicLibrary.dll" "$(SolutionDir)"
-cd "$(SolutionDir)"</PostBuildEvent>
+copy "$(TargetPath)" "$(SolutionDir)mods/ra/"</PostBuildEvent>
   </PropertyGroup>
```
Comment (obrakmann) [inline OpenRA.Mods.RA/OpenRA.Mods.RA.csproj]: This causes the build with MonoDevelop to fail, see https://github.com/OpenRA/OpenRA/pull/7422#issuecomment-72504436

**Comment by obrakmann** [comment]
I'd also ask you to remove the instructions that copy the fuzzy logic lib from the `PostBuildEvent` in OpenRA.Mods.Common.csproj. This PR would then cleanly supersede #7422.

**Comment by ScottNZ** [comment]
These fuzzy changes don't have anything to do with "Move the AI to Mods.Common", so whether it cleanly supercedes #7422 is debatable ;)

**Comment by obrakmann** [comment]
Since the AI is the only user of the fuzzylogic library I wouldn't say that they "don't have anything to do with" this move at all. Quite the contrary, they're needed here to ensure the thing still compiles.

**Comment by ScottNZ** [comment]
I disagree, but instead of bikeshedding let's get it merged.

File: OpenRA.Mods.Common/OpenRA.Mods.Common.csproj [hunk 2]
```diff
@@ -25,12 +25,17 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <CustomCommands>
       <CustomCommands>
+        <Command type="AfterBuild" command="cp ../thirdparty/FuzzyLogicLibrary.dll ../" workingdir="${ProjectDir}" />
```
Comment (ScottNZ) [inline OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]: This should be unnecessary.

**Comment by penev92** [comment]
Now this should do it.

**Comment by obrakmann** [comment]
yep. :+1:

**Comment by Mailaender** [comment]
Tested 3 mods and Makefile as well as MonoDevelop compile after `make clean`. Confirmed that this still works. :white_check_mark: Thanks for cleaning this up.

**Pull request merged**